tests/s3mock.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+from networktest.mock import HttpApiMock, HttpApiMockEndpoint
+
+
+class S3Mock(HttpApiMock):
```

Comment:
I think it'd probably be fine if this just lived in the `mock` module

Reply:
Done.